actors/util/adt/map.go

```diff
@@ -90,3 +90,15 @@ func (h *Map) Delete(k Keyer) error {
 	h.root = newRoot
 	return nil
 }
+
+// ForEach applies fn to each key value in hamt.
+func (h *Map) ForEach(fn func(key string, v interface{}) error) error {
```

Comment:
Use the CBORUnmarshalable interface from runtime (and replace the cbg references above with similar).

Reply:
If I we do that here the hamt `ForEach` code won't accept it (unless there is some type-foo that can help here?). WIll do it for all other methods in this file.